app/controllers/items_controller.rb

```diff
@@ -1,28 +1,58 @@
 class ItemsController < ApplicationController
-    def index
-        @items=Item.all
-    end
-    
-    def show
-        @item=Item.find(params[:id])
-    end
-    
-    def new
-        @item=Item.new
-        @facilities=Facility.all
-    end
-    
-    def create
-        @item=Item.new
-        @item.name=params[:item][:name].upcase
-        @item.date=params[:item][:date]
-        @item.description=params[:item][:description]
-        @item.price=params[:item][:price]
-        @item.status=params[:item][:status]
-        @item.facility_id=params[:facility_id]
-        @item.status="pending"
-        @item.save
-        redirect_to items_path
-    end
+  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
+  
+  def index
+    @items=Item.where("status = 'approved'")
+  end
+    
+  def show
+    @item=Item.find(params[:id])
+  end
+    
+  def new
+    @item=Item.new
+  end
+    
+  def create
+    @item = Item.create(create_params)
+    @item.status = "pending"
+    @item.save
+    redirect_to "/"
```

Comment:
use the path helper method `root_path`

Reply:
Changed / to helper methods in admins, items, facilities and procedures controllers.